var/spack/repos/builtin/packages/py-storm/package.py

```diff
@@ -9,12 +9,15 @@
 class PyStorm(PythonPackage):
     """Storm is an object-relational mapper (ORM) for Python"""
     homepage = "https://storm.canonical.com/"
-    url      = "https://launchpad.net/storm/trunk/0.20/+download/storm-0.20.tar.bz2"
+    pypi     = 'storm/storm-0.25.tar.gz'
 
-    version('0.23', sha256='01c59f1c898fb9891333abd65519ba2dd5f68623ac8e67b54932e99ce52593d3')
-    version('0.20', sha256='1fe016c9ec40520eafc3cf359f1ec2b7fa86be91e45c9279bfb0ea3b06390a82')
+    version('0.25', sha256='ec7cc8897638f94f6b75c6a2af74aa9b31f5492d7a2f9482c08a8dd7b46adb14')
+    version('0.23', sha256='01c59f1c898fb9891333abd65519ba2dd5f68623ac8e67b54932e99ce52593d3',
+            url='https://launchpad.net/storm/trunk/0.23/+download/storm-0.23.tar.bz2')
```

Comment:
Why not use PyPI for 0.23?
```suggestion
            url='https://files.pythonhosted.org/packages/source/s/storm/storm-0.23.tar.bz2')
```